packages/contracts-bedrock/src/L1/SystemConfigInterop.sol

```diff
@@ -65,9 +65,9 @@ contract SystemConfigInterop is SystemConfig {
         Storage.setAddress(DEPENDENCY_MANAGER_SLOT, _dependencyManager);
     }
 
-    /// @custom:semver +interop
+    /// @custom:semver +interop-beta.0
     function version() public pure override returns (string memory) {
-        return string.concat(super.version(), "+interop");
+        return string.concat(super.version(), "+interop-beta.0");
```

Comment:
what version should be used here @tynes ?

Reply:
The interop devnets are not upgradable so the versioning doesn't matter a ton. We just want the linting to pass, which seems to be buggy around the inherited contracts, forcing upgrades in the version number

Reply:
Correct version is `interop.1`